tools/convert_bigfile_gadget_format3.py

```diff
@@ -0,0 +1,271 @@
+"""
+Script to convert between MP-Gadget formatted bigfile snapshots and Gadget-3 formatted HDF5 snapshots.
+Both directions are supported.
+
+This is to make it easier for simulators to start using MP-Gadget.
+
+Known name changes are accounted for: for example MP-Gadget has "Position" and Gadget-3 has "Coordinates".
+
+No unit conversion is done! Be careful with older Gadgets which do not store units in the snapshot header.
+
+TODO: Support Black hole data arrays better.
+"""
+
+from __future__ import print_function
+import argparse
+import os
+import os.path
+import re
+import glob
+import bigfile
+import h5py
+import numpy as np
+
+
+class NameMaps(object):
+    """Maps names between HDF5 and Bigfile."""
+    def __init__(self):
+        #Map of (only changed) block names between HDF5 and bigfile snapshots.
+        self.hdf_to_bigfile_map = { "Coordinates" : "Position",
+                                    "Velocities": "Velocity", "Masses": "Mass",
+                                    "NeutralHydrogenAbundance": "NeutralHydrogenFraction",
+                                    "GFM_Metallicity": "Metallicity",
+                                    "ParticleIDs" : "ID",
+                                  }
+        #This requires python 2.7
```

Comment:
? Python 3?

Reply:
Er, yes, indeed. But I think we can assume python 3 nowdays anyway, so I'll just remove the comment (and the from __future__).